Added Support for Different Size Cells

the-mentor opened the pull request:
This is limited in a grid of 12

### Description of the Change
This change allows the user to specify a size when creating a New-PodeWebCell in a grid of 12 as bootstrap allows here [bootstrap docs](https://getbootstrap.com/docs/4.0/layout/grid/#all-breakpoints)

### Examples
```
Start-PodeServer {
    Add-PodeEndpoint -Address localhost -Port 8888 -Protocol Http

    Use-PodeWebTemplates -Title 'Example' -Theme Light

    Add-PodeWebPage -Name 'Services' -NoTitle -Icon 'Settings' -ScriptBlock {
        New-PodeWebGrid -Cells @(
            New-PodeWebCell -Size 8 -Content @(
                New-PodeWebCard -Content @(New-PodeWebParagraph -Value 'Size 8')
            )
            New-PodeWebCell -Size 4 -Content @(
                New-PodeWebCard -Content @(New-PodeWebParagraph -Value 'Size 4')
            )
        )
    }
}
```

Without the change:
![image](https://user-images.githubusercontent.com/11185446/121215056-56111680-c888-11eb-9642-363e0ed4718a.png)


With the change:
![image](https://user-images.githubusercontent.com/11185446/121214979-4560a080-c888-11eb-94ec-645c6ad0488d.png)

Badgerati commented on src/Public/Layouts.ps1:
```diff
@@ -33,7 +33,8 @@ function New-PodeWebCell
         [hashtable[]]
-        $Content
+        $Content,
+        [ValidateRange(1, 12)][int]$Size
```
Split this one onto separate lines, with a Parameter attribute (so it's consistent with the rest):

```powershell
[Parameter()]
[ValidateRange(1, 12)]
[int]
$Size
```

I'm wondering if calling the parameter `$Width` might be more fitting, for if we ever need to add a `$Height` at some point 🤔

the-mentor replied:
I'll fix this tomorrow and renaming it to $Width sounds like a good idea so I'll do that as well.

the-mentor commented:
@Badgerati I've fixed the formatting and renamed the parameter to Width.

Unless there are any other changes you would like this can probably get merged.

Badgerati reviewed (approved):
LGTM!
